### ISSUE TITLE
CI should check that we don't build `rustc_codegen_spirv` more than once.

### ISSUE THREAD

**Issue by eddyb** [EmbarkStudios/rust-gpu#549]
See #546 for some context and ideas - especially https://github.com/EmbarkStudios/rust-gpu/pull/546#issuecomment-809089160.

Sadly, this is likely blocked on moving CI to use `--release` to build examples (which depend on "build dependency release mode `rustc_codegen_spirv` via the `spirv-builder`-using build scripts), otherwise the problem can't be reproduced in the first place (AFAIK).

**Comment by oisyn** [comment]
Superseded by #911
